### DISCUSSION TITLE
Duplicate records after synchronize from QFieldCloud

### DISCUSSION THREAD

**Discussion by geomondego** [opengisch/QField#3015] [Bugs]
Hello,

This morning i was testing a project shared with 18 users.
Each user created one point, uploaded it to QFieldCloud and then synced again to get all the points scored by the 18 users.
Several points now show several duplicate records. Two, four, six, etc. Image below

Any idea what might have happened?

Regards,
Nelson

![Screenshot_2022-06-27-11-03-04-815_ch opengis qfield](https://user-images.githubusercontent.com/4389239/175918340-d7d47caa-ec09-4521-8fc3-cd46185f3d79.jpg)

**Comment by lucienicolier** [comment]
Hi @geomondego,

We are going to have a look at this issue, but we do not know exactly when.

If it is urgent, please contact info@opengis.ch.

Best regards

**Reply by geomondego** [reply]
@lucienicolier,
Thanks. It is not urgent. 

Regards,
Nelson

**Reply by rungdung** [reply]
I came across the same issue. We had around 20 contributors and over 70 duplicates had been created in a total of 280 features mapped. We only realised after sharing the data and checked for duplicate geometries in QGIS.

**Reply by ZsanettMed** [reply]
Hi, We are aware of this issue and working on a fix, which will be published soon. We will keep you informed. 

We apologize for the inconvenience!

Best regards, 
Zsanett

**Comment by sigiot** [comment]
Hello, we have the same problem when several contributors submit the changes in the same time to cloud, do you found a solution from that problem.
